docs/user-guide/ci.rst

```diff
@@ -0,0 +1,74 @@
+.. role:: bash(code)
+   :language: bash
+
+CI Configuration
+================
+
+Follow these steps to configure CI for GitHub Actions on your repository:
+
+* Step 1:
+   Add the Certora CLI key `as a secret`_ to your repository, the secret name is "CERTORAKEY"
+   and the value is the key provided.
+
+* Step 2:
+   Create a :bash:`certora_verification.yml` file under :bash:`.github` directory and 
+   use the following `.yaml` example as a template you can use to start running `certora-cli`.
+
+.. code-block:: yaml
+
+   name: certora-verification
+
+   on:
+   push:
+      branches:
+         - main
+   pull_request:
+      branches:
+         - main
+
+   workflow_dispatch:
+
+   jobs:
+      verify:
+         runs-on: ubuntu-latest
+
+         steps:
+            - uses: actions/checkout@v4
+            with:
+               submodules: recursive
+
+            - name: Install python
+            uses: actions/setup-python@v2
+            with: { python-version: 3.9 }
+
+            - name: Install java
+            uses: actions/setup-java@v1
+            with: { java-version: "11", java-package: jre }
+
+            - name: Install certora cli
+            run: pip3 install certora-cli==7.0.7
+
+            - name: Install solc
+            run: |
+               wget https://github.com/ethereum/solidity/releases/download/v0.8.23/solc-static-linux
+               chmod +x solc-static-linux
+               sudo mv solc-static-linux /usr/local/bin/solc8.23
+
+            - name: Verify ${{ matrix.rule }}  
+            env:  
+               CERTORAKEY: ${{ secrets.CERTORAKEY }}  
+            run: 
+               certoraRun  ${{ matrix.rule }}  
+
+      strategy:  
+         fail-fast: false  
+         max-parallel: 16  
+         matrix:  
+            rule:  
```

Comment:
let's put a link here to `--rule` and `--exclude_rule` docs:
https://docs.certora.com/en/latest/docs/prover/cli/options.html#rule-rule-name
https://docs.certora.com/en/latest/docs/prover/cli/options.html#exclude-rule-rule-name-pattern

Reply:
@shellygr I couldn't use these as a link inside of a code block.
I'm suggesting writing it at the end of the page, for example:

**Further reading**
`--rule`: https://docs.certora.com/en/latest/docs/prover/cli/options.html#rule-rule-name
`--exclude_rule` https://docs.certora.com/en/latest/docs/prover/cli/options.html#exclude-rule-rule-name-pattern

Reply:
I meant not as link but as part of the yml, but that's also ok. Maybe say it's also related to building the matrix

Reply:
got it, i added it next to them, let me know if its looks allright